docs/docs/FAQ.md

```diff
@@ -14,7 +14,11 @@ sidebar_position: 7
 
 ### How can I sync an existing directory with Immich's server?
 
-Immich doesn't have the mechanism to sync an existing directory with the server. There is however, a helper CLI tool to help you bulk upload the existing photos and videos to the server. You can find the guide to use the CLI tool [here](/docs/features/bulk-upload.md).
+Immich doesn't have two-way synchronization ([yet](https://github.com/immich-app/immich/discussions/1006)), but the [command line tool](/docs/features/bulk-upload.md) can bulk upload items from a directory to Immich.
+
+### Why doesn't Immich watch an existing photo gallery directory?
+
+The initial approach of Immich is to become a backup tool, primarily for mobile device usage. Thus, all the assets must be uploaded from the mobile client. The app was architectured to perform that job well, however contributions are welcome to add more features.
```

Comment:
The "contributions welcome" part in this context seems to imply that a PR for folder watching is welcome, which I think isn't the case? @alextran1502 ?

Reply:
That's probably not worded right, let me fix